code/game/objects/items/weapons/cards_ids.dm

```diff
@@ -343,8 +350,19 @@
 	name = "agent card"
```

Comment:
TBH, this agent card stuff is big enough to be in its own file. Could you move it there please?